app/src/pages/playground/SpanPlaygroundPage.tsx

```diff
@@ -26,12 +29,55 @@ export function SpanPlaygroundPage() {
   );
 
   return (
-    <Playground
-      instances={
-        playgroundInstance != null
-          ? [playgroundInstance]
-          : [createPlaygroundInstance()]
-      }
-    />
+    <Flex direction="column" height="100%">
+      <SpanPlaygroundBanners span={span} />
+      <Playground
+        instances={
+          playgroundInstance != null
+            ? [playgroundInstance]
+            : [createPlaygroundInstance()]
+        }
+      />
+    </Flex>
+  );
+}
+
+function SpanPlaygroundBanners({
+  span,
+}: {
+  span: Extract<
+    NonNullable<spanPlaygroundPageLoaderQuery$data["span"]>,
+    { __typename: "Span" }
+  >;
+}) {
+  const navigate = useNavigate();
+  const [showBackBanner, setShowBackBanner] = useState(true);
+  return (
+    <div>
+      {showBackBanner && (
+        <Alert
+          variant="info"
+          title="LLM Span Replay"
+          banner
+          dismissable
+          onDismissClick={() => {
+            setShowBackBanner(false);
+          }}
+          extra={
+            <Button
+              variant="default"
+              icon={<Icon svg={<Icons.ArrowBack />} />}
+              onClick={() => {
+                navigate(
+                  `/projects/${span.project.id}/traces/${span.context.traceId}?selectedSpanNodeId=${span.id}`
+                );
+              }}
+            >
+              Back to Trace
+            </Button>
+          }
+        >{`Replay and iterate on your LLM call from your ${span.project.name} project`}</Alert>
```

Comment:
curious about the project part here since playground is sitting to the side of any project. is this referring to playground spans or the original spans project?

Reply:
the original project!